### PR TITLE
task/Add Sandbox note to release calendar

### PR DESCRIPTION
**Pull request by RichardEustace**

### CODE REVIEW

File: docs/release-calendar/index.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 layout: default
 title: Release Calendar
 ---
- 
+
 # Release Calendar
 {: .govuk-heading-xl}
 
+**NOTE:** Note: From the 01/09/2020 the Sandbox release refers to the new Sandbox environment (UI URL: https://www.sandbox.manage-roadworks.service.gov.uk and  API URL: https://api.sandbox.manage-roadworks.service.gov.uk) and going forward this is the only sandbox environment that will recieve updates.
```
Comment (cpaul92) [inline docs/release-calendar/index.md:9]:
```suggestion
**NOTE:** From the 01/09/2020 the Sandbox release refers to the new Sandbox environment (UI URL: https://www.sandbox.manage-roadworks.service.gov.uk and API URL: https://api.sandbox.manage-roadworks.service.gov.uk) and going forward this is the only sandbox environment that will receive updates.
```

**Review by caoimheb — APPROVED** [review]
Approved pending the changes in @cpaul92 's comment

File: docs/release-calendar/index.md [hunk 2]
```diff
@@ -2,10 +2,12 @@
 layout: default
 title: Release Calendar
 ---
- 
+
 # Release Calendar
 {: .govuk-heading-xl}
 
+**NOTE:** From the 01/09/2020 the Sandbox release refers to the new Sandbox environment (UI URL: https://www.sandbox.manage-roadworks.service.gov.uk and  API URL: https://api.sandbox.manage-roadworks.service.gov.uk) and going forward this is the only sandbox environment that will receive updates.
```
Comment (cpaul92) [inline docs/release-calendar/index.md:9]:
```suggestion
**NOTE:** From the 01/09/2020 the Sandbox release refers to the new Sandbox environment (UI URL: https://www.sandbox.manage-roadworks.service.gov.uk and API URL: https://api.sandbox.manage-roadworks.service.gov.uk) and going forward this is the only sandbox environment that will receive updates.
```

**Review by cpaul92 — APPROVED** [review]
Approved pending removal of double space

**Pull request merged**